numpy/core/shape_base.py

```diff
@@ -177,6 +177,71 @@ def atleast_3d(*arys):
         return res
 
 
+def atleast_nd(n, *arys):
+    """
+    View inputs as arrays with at least `n` dimensions.
+
+    Parameters
+    ----------
+    n : scalar
```

Comment:
But almost all numpy functions have the array on which the function works first in the list. The only real reason for doing it differently here is that one is allowed to pass in multiple arrays as `*args` (a design mistake, in my opinion, but too late...).

Anyway, these are just my opinions; as I'm not sure I understand the need for the function, I probably should not be the one arguing and rather leave it to the numpy maintainers to decide.